bindings/protocols/profinet/index.html

```diff
@@ -0,0 +1,1040 @@
+        </p>
+    </section>
+    <section id='introduction'>
+        <h2>Introduction</h2>
+        <p>
+            PROFINET is an open standard communication protocol developed and published by PROFIBUS Nutzerorganisation (PNO) for Industrial Ethernet. 
+            PROFINET which is an Ethernet version of PROFIBUS is designed to satisfy requirements of most industrial automation processes by providing a 
+            high-speed data transfer for time critical applications like motion control and drive. 
+            It can also have a safety (PROFIsafe) implementation onboard for safety related application. 
+        </p>
+        <p>
+            Generally, profinet is classified into PROFINET IO (Input Output) which is used for distributed input/output processes of field devices (sensors and actuators) 
+            data and PROFINET CBA (Component Based Automation) which is used for distributed automation. The scope of this document focuses on the PROFINET IO. 
+            PROFINET IO network consist of three main actors which are IO controller, IO supervisor and IO device. 
+            The IO controllers are PLCs and DCS, they are most used for profinet cyclic communications like IO data exchange. 
+            The IO supervisor is a standalone application that is mostly used for acyclic communications like diagnostics and parameterization. 
+            The IO device is general field devices that are controlled by both IO controller and IO supervisor.
+        </p>
+        <p>
+            Due to the different requirements by different industrial automation applications, PROFINET IO provides three different communications channels to exchange data between IO controllers and IO device or IO supervisor and IO devices. 
+            The Real-Time (RT) and Isochronous Real Time (IRT) channels are used for cyclic data transfer and high-speed cyclic data transfer for motion control application respectively, 
+            the User Data Protocol (UDP) channel is mostly used by IO supervisor for parameterization, diagnostic and acyclic communications. 
+            The addressing in PROFINET slot/sublsot/index mechanism where the slot identifies a module and subslot identifies submodules that exist within a module. The index identifies datapoint of interest in the submodule.
+        </p>
+        <p>
+            Concretely, this document describes how WoT TDs can be used to describe devices that use the PROFINET acyclic communication (IO supervisor). 
+            The IO controller operations that use cyclic communication is beyond the scope of this document. This document explains how to create valid URLs and Forms for read or write operation from or to an IO device respectively. 
+            Developers are encouraged to use this document as an implementation guidelines and as a reference for the creation of their own binding implementations. The following sections will cover the URL format, the vocabulary and a list of Form examples.
```

Comment:
In addition to @danielpeintner's suggestion
```suggestion
            Developers are encouraged to use this document as an implementation guidelines and as a reference for the creation of their own binding implementations. The following sections will cover the URL format, the vocabulary, and a list of Form examples.
```